arrows/matlab/mxtypes.h

```diff
@@ -85,26 +84,27 @@ struct MxIntTy<T, typename std::enable_if<
     !MxLogicalTy<T>::value &&
     !MxCharTy<T>::value,
     T>::type> : std::true_type {};
-/** Traits for arithmetic types.
- */
+/// Traits for arithmetic types.
 template <typename T, typename U = T>
 struct MxArithmeticTy : std::false_type {};
 template <typename T>
 struct MxArithmeticTy<T, typename std::enable_if<
     (std::is_floating_point<T>::value) || (MxIntTy<T>::value),
     T>::type> : std::true_type {};
 
-/**********************************************/
-/* Introducing traits for MATLAB array types. */
-/**********************************************/
+//**********************************************
+///
+/// Introducing traits for MATLAB array types.
+///
+//**********************************************
+
```

Comment:
Likewise. (And again, an extra blank line.)

Reply:
Fixed.